api/src/main/java/org/apache/iceberg/events/IncrementalScanEvent.java

```diff
@@ -60,4 +62,8 @@ public Expression filter() {
   public Schema projection() {
     return projection;
   }
+
+  public boolean fromSnapshotInclusive() {
```

Comment:
Would it be more clear for this to be `isFromSnapshotInclusive`?

Reply:
agree. updated